lib/iris/fileformats/nimrod.py

```diff
@@ -87,46 +89,95 @@
     "sat_space_count",
     "ducting_index",
     "elevation_angle",
+    "neighbourhood_radius",
+    "threshold_vicinity_radius",
+    "recursive_filter_alpha",
+    "threshold_fuzziness",
+    "threshold_duration_fuzziness",
+    "data_header_float32_05",
+    "data_header_float32_06",
+    "data_header_float32_07",
+    "data_header_float32_08",
+    "data_header_float32_09",
+    "data_header_float32_10",
+    "data_header_float32_11",
+    "data_header_float32_12",
+    "data_header_float32_13",
+    "data_header_float32_14",
+    "data_header_float32_15",
+    "data_header_float32_16",
+    "data_header_float32_17",
+    "data_header_float32_18",
+    "data_header_float32_19",
+    "data_header_float32_20",
+    "data_header_float32_21",
+    "data_header_float32_22",
+    "data_header_float32_23",
+    "data_header_float32_24",
+    "data_header_float32_25",
+    "data_header_float32_26",
+    "data_header_float32_27",
+    "data_header_float32_28",
+    "data_header_float32_29",
+    "data_header_float32_30",
+    "data_header_float32_31",
+    "data_header_float32_32",
 )
 
 
 # data specific header (int16) elements
 data_header_int16s = (
-    "radar_num",
-    "radars_bitmask",
-    "more_radars_bitmask",
-    "clutter_map_num",
-    "calibration_type",
-    "bright_band_height",
-    "bright_band_intensity",
-    "bright_band_test1",
-    "bright_band_test2",
-    "infill_flag",
-    "stop_elevation",
-    "int16_vertical_coord",
-    "int16_reference_vertical_coord",
-    "int16_y_origin",
-    "int16_row_step",
-    "int16_x_origin",
-    "int16_column_step",
-    "int16_float32_mdi",
-    "int16_data_scaling",
-    "int16_data_offset",
-    "int16_x_offset",
-    "int16_y_offset",
-    "int16_true_origin_latitude",
-    "int16_true_origin_longitude",
-    "int16_tl_y",
-    "int16_tl_x",
-    "int16_tr_y",
-    "int16_ty_x",
-    "int16_br_y",
-    "int16_br_x",
-    "int16_bl_y",
-    "int16_bl_x",
-    "sensor_id",
-    "meteosat_id",
-    "alphas_available",
+    "threshold_type",
```

Comment:
It seems peculiar to have discarded all the previous names for these elements + replaced them with totally different interpretations, but I guess you know what you are doing !

Reply:
While the old versions are still mentioned in the Nimrod-format documentation, I'm not aware of them being used any more (Radarnet has moved over to HDF5) and they have all been overloaded with new definitions. This is the "data-specific" section of the header. While it might be possible to devise some logical test based on the field_id, I don't have any data that uses the old values to test it with.
The decision to remove the old definitions entirely is based on my confidence that I've covered all the use-cases for Nimrod-format data without them.

Reply:
Ok, thanks for explaining!